packages/transaction-controller/src/TransactionController.ts

```diff
@@ -188,6 +191,10 @@ export class TransactionController extends BaseController<
 
   private readonly mutex = new Mutex();
 
+  private readonly getAdvancedGasFeeByChainId: (
```

Comment:
`getSavedGasFees`?

I worry `ByChainId` implies the result is a map of values for different chains.

Reply:
Renamed this